### DISCUSSION TITLE
[Feature Request] Audio Waveform Area

### DISCUSSION THREAD

**Discussion by vpoguru** [GrandOrgue/LoopAuditioneer#4] [Ideas]
The ability to zoom the audio waveform horizontally would be highly beneficial, as the current method of resizing the window is limited by the monitor’s size. This feature would greatly improve the precision of setting CUE points.

Additionally, implementing an option to scale the waveform’s amplitude to a specific value, such as 50% of the window height, would be a valuable time-saver. This feature would be particularly helpful when working with quieter sections of audio, where manual zooming is currently required.

For further improvement, enabling both vertical and horizontal zoom using the mouse wheel would enhance usability — for example, vertical zoom by default, and horizontal zoom when holding the Shift key.

**Comment by larspalo** [comment]
> The ability to zoom the audio waveform horizontally would be highly beneficial, as the current method of resizing the window is limited by the monitor’s size. This feature would greatly improve the precision of setting CUE points.

Well, I think that a modified waveform overlay/magnification window will take care of that as I outlined in #7. I'd rather keep the main window waveform simple and useful for the general sample overview.

> Additionally, implementing an option to scale the waveform’s amplitude to a specific value, such as 50% of the window height, would be a valuable time-saver. 

I've added this as a TODO. But I'd implement it as a View option to scale the waveform to at least 50% (if at all possible) automatically when a file is opened.
